mettle/extensions/lua/lua.c

```diff
@@ -0,0 +1,104 @@
+#include "extension.h"
+
+#include <lua.h>
+#include <lauxlib.h>
+#include <lualib.h>
+
+#include <string.h>
+#include <stdlib.h>
+
+#include "lua.h"
+
+char *str_output = NULL;
+
+/*
+ * Override 'print' so the extension won't crash
+ * Thanks @timwr for the link: https://gist.github.com/5at/3671566
+ */
+static int print_to_framework_side(lua_State *lua) {
+	int nargs = lua_gettop(lua);
+	for (int i = 0; i < nargs; i++) {
+		const char *str = lua_tostring(lua, i);
+
+        // Extend buffer
+		str_output = realloc(str_output, strlen(str_output) + strlen(str));
```

Comment:
This is really pedantic but calling strlen within a loop seems inefficient. Let me send you a PR where we keep track of the length.

Reply:
Ok, I was trying something with `Lua_strlen` on my laptop, let's compare our changes! (and sorry for the code formatting, `vim` doesn't like two space indentation on my side, probably just a config away anyways).